tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -757,6 +757,11 @@ SpirvEmitter::getInterfacesForEntryPoint(SpirvFunction *entryPoint) {
   for (auto *moduleVar : spvBuilder.getModule()->getVariables()) {
     if (moduleVar->getStorageClass() != spv::StorageClass::Input &&
         moduleVar->getStorageClass() != spv::StorageClass::Output) {
```

Comment:
Is this only a problem for ray tracing? What happens if we have multiple fragment shaders in a `lib`?

Reply:
I believe it's only a problem for ray tracing, since other shader types keep track of their stage variables in the `stageVars` vector, but ray tracing shaders do not – https://github.com/microsoft/DirectXShaderCompiler/blob/30f45bcf0f56fe675d797bde700891f4a6a816c3/tools/clang/lib/SPIRV/DeclResultIdMapper.cpp#L4524.